### PR TITLE
feat(icons): add normalize action

### PR DESCRIPTION
**Pull request by mloussot**
**What is the problem this PR is trying to solve?**
Symbolize the Normalize component used notably in DataStream

**What is the chosen solution to this problem?**
Create a normalize icon

**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by mloussot** [comment]
In DataStream the icon has to be colored in blue tones.
Here are the colors to use : 
To replace #000 use #688ba8
To replace #515151 use #7297b8
To replace #bcbcbc use #b9cade
To replace #8c8c8c use #9fb8d2

File: packages/icons/src/normalize.svg [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <g fill="none" fill-rule="evenodd" transform="translate(2.56 -.656)">
+    <g fill-rule="nonzero" transform="translate(7 13)">
+      <polygon fill="#000" points=".357 3.655 3.688 3.655 3.688 .323 .357 .323"/>
```
Comment (jmfrancois) [inline packages/icons/src/normalize.svg]: I'm not sure this is safe to fill with color here. Please use class + css

Comment (jmfrancois) [inline packages/icons/src/normalize.svg]: CF to beam.svg icon

Comment (acateland) [inline packages/icons/src/normalize.svg]: style properties like fill should be set up on a g element along with a css class wich allow us to target element.

**Pull request merged**